src/App.js

```diff
@@ -37,28 +37,60 @@ const NoLogin = () => {
  * Main App component. Wrap the main react-router components together with
  * the various dialogs and error messages that may be needed.
  */
-const App = ({ history, config, appReady }) => {
-  if (!config.get('loginToken')) { // login is missing
+/* const App = ({ history, config, appReady }) => {
+
+} */
```

Comment:
git remembers what is was before, no need to keep that comment

Reply:
done